crates/net/network/src/metrics.rs

```diff
@@ -32,3 +32,17 @@ pub struct NetworkMetrics {
     /// Number of invalid/malformed messages received from peers
     pub(crate) invalid_messages_received: Counter,
 }
+
+/// Metrics for the TransactionManager
+#[derive(Metrics)]
+#[metrics(scope = "network")]
+pub struct TransactionsManagerMetrics {
+    /// Total number of propagated transactions
+    pub(crate) propagated_transactions: Counter,
+
+    /// Total number of new peers
+    pub(crate) new_peers: Counter,
+
+    /// Total number of lost peers
+    pub(crate) lost_peers: Counter,
```

Comment:
these are only secondary for the tx manager, since the tx manager merely consumes the corresponding events (established/disconnect).

So we don't shouldn't set them from tx manager and can remove them here.

Reply:
Ok I will get it done.